modules/core/core-api/src/main/java/com/enonic/xp/api/ApiMount.java

```diff
@@ -0,0 +1,13 @@
+package com.enonic.xp.api;
+
+import java.util.Locale;
+
+public enum ApiMount
+{
+    GLOBAL, ANY_SITE_ENDPOINT, SITE_ENDPOINT, ANY_PROJECT_ENDPOINT, PROJECT_ENDPOINT, ANY_WEBAPP_ENDPOINT, WEBAPP_ENDPOINT, ADMIN_ENDPOINT;
+
+    public static ApiMount from( final String value )
+    {
+        return value != null ? ApiMount.valueOf( value.toUpperCase( Locale.ROOT ) ) : null;
```

Comment:
why making it null save? we are reading it from valid XML
why returning null - this will make user's code responsible to check for null